### ISSUE TITLE
Configurations' name

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework7G2#27]
This paper will let you know some important configurations for our task that will aid to understand the structure type.
Source:
J. P. Rose and R. S. Berry, ‘‘Towards Elucidating the Interplay of Structure and Dynamics in Clusters: Small KCl Clusters as Models,’’ J. Chem. Phys. 96, 517–538 ~1992!.
![configurations_name](https://github.com/ubsuny/23-Homework7G2/assets/143649367/6617cbb3-8356-4ad8-81e3-8482dd17d886)

**Comment by s4il3sh** [comment]
yep. figure matches with our 5th fig. but fig. d does it match with figure 4? figure 4 looks like hexagonal

**Comment by WildJimmy** [comment]
This is interesting and very helpful, thank you
